lib/BridgedRoom.js

```diff
@@ -91,6 +91,18 @@ BridgedRoom.prototype.unlinkMatrixRoom = function(matrixRoomId) {
     ]);
 };
 
+BridgedRoom.prototype.swapRoomIds = function(oldRoomId, newRoomId) {
+    if (this._portalMatrixRoomId === oldRoomId) {
```

Comment:
If this is ever false it would be Very Bad, right? We should log in an `else`

Reply:
Well, it could also be a plumbed room.